Source/Csla.Blazor/ViewModel.cs

```diff
@@ -236,15 +237,11 @@ public async Task SaveAsync()
       {
         if (Model is Core.ITrackStatus obj && !obj.IsSavable)
         {
-          if (obj.IsBusy)
+          if (obj is INotifyBusy notifyBusy)
```

Comment:
This test is unnecessary since `ITrackStatus` guaranetees obj is of that type.

Reply:
ok. I will update that.